cg/store/api/findbusinessdata.py

```diff
@@ -384,8 +385,18 @@ def samples_by_ids(self, **identifiers) -> Query:
     def get_sample_by_name(self, name: str) -> Sample:
         return self.Sample.query.filter(Sample.name == name).first()
 
+    def get_samples_by_type(self, case_id: str, sample_type: SampleType) -> Optional[List[Sample]]:
+        """Extract samples given a tissue type."""
+        samples: Query = self.Sample.query.join(Family.links, FamilySample.sample)

```

Comment:
If you have time - it would be great to separate the query statement into a separate function in status.py. Much like
```
    def _get_sample_query(self) -> Query:
        """Return sample query."""
        return self.query(Sample)
```
But of course reflect that this sample query contains `Family` and `FamilySample` as well.

Reply:
Good idea 👍

Reply:
It's a good idea @henrikstranneheim, but I'm not sure how to proceed with it... 

We already have a newly added `_get_case_sample_query` in this file, which I suppose has been included here because of how we have structured `Store` and `CoreHandler`, i.e. not allowing methods to be used between these StatusHandler, FindBusinessDataHandler, etc. classes.

To bypass it, we could create some inheritances between the handlers, having StatusHandler as a parent class for example, but I'm not sure how this would affect the Store class and if it would make sense to have both implementations.

Reply:
Sorry my bad, I meant that you could add the new method within the same module i.e in `findbusinessdata.py`